examples/function_router/src/bin/function_router.rs

```diff
@@ -4,4 +4,6 @@ fn main() {
     wasm_logger::init(wasm_logger::Config::new(log::Level::Trace));
     #[cfg(feature = "csr")]
     yew::Renderer::<App>::new().render();
+    #[cfg(not(feature = "csr"))]
+    panic!("You must enable the csr feature to run this binary");
```

Comment:
```suggestion
    compile_error!("You must enable the csr feature to run this binary");
```
Seems like a better idea to have a compilation error

Reply:
Further digging reveals that `Cargo.toml` has a `required-features = ["yew/csr"]` field for binaries  that can be used. Even better!